File: pulumi/examples/typescript/digitalocean_loadbalancer_example.ts

# Pulumi DigitalOcean Load Balancer Example (Python)
import pulumi
import pulumi_digitalocean as digitalocean

example_lb = digitalocean.LoadBalancer(
    "example-lb",
    region="nyc3",
    forwarding_rules=[{
        "entry_protocol": "http",
        "entry_port": 80,
        "target_protocol": "http",
        "target_port": 80
    }]
)

pulumi.export("load_balancer_id", example_lb.id)

File: pulumi-infra-project/pulumi/examples/python/digitalocean/digitalocean_loadbalancer_example.py
"""
Pulumi DigitalOcean Load Balancer Example (Python)

Setup:
  pip install pulumi pulumi_digitalocean
Usage:
  pulumi up
"""
import pulumi
import pulumi_digitalocean as do

lb = do.LoadBalancer('example-lb',
    region='nyc3',
    forwarding_rules=[do.LoadBalancerForwardingRuleArgs(
        entry_protocol='http',
        entry_port=80,
        target_protocol='http',
        target_port=80
    )]
)
pulumi.export('loadbalancer_id', lb.id)
